### PR TITLE
Rd 2883

### PR DESCRIPTION
**Pull request by Ivneetwahi**

### CODE REVIEW

File: jenkins/release/Finish [hunk 1]
```diff
@@ -1,6 +1,17 @@
...
+@Library('reanplatform@RD-2883') _
+node
+{
+  try
+  {
+    gemPipelines.releaseFinish(name: 'awesome_gem')
+  }
+  catch(err) 
+  {
+  	if (env.mailAlert=="true")
```
Comment (lf12345) [inline jenkins/release/Finish:10]: Shouldn't it be ${env. mailAlert}?

File: jenkins/release/Finish [hunk 2]
```diff
@@ -1,6 +1,17 @@
...
+	    {
+  	    mail to: env.RECIPIENTS,
```
Comment (lf12345) [inline jenkins/release/Finish:12]: Shouldn't it be ${env. RECIPIENTS}?

File: jenkins/hotfix/Build [hunk 3]
```diff
@@ -1,4 +1,4 @@
-@Library('reanplatform') _
+@Library('reanplatform@RD-2883') _
```
Comment (lf12345) [inline jenkins/hotfix/Build:1]: Please remove the rean library branch since we aren't using it at the moment.

File: lib/awesome_gem/version.rb [hunk 4]
```diff
@@ -2,5 +2,5 @@
 
 module AwesomeGem
   # Version Number
-  VERSION = '1.28.0'
+  VERSION = '1.30.0'
```
Comment (lf12345) [inline lib/awesome_gem/version.rb:5]: I think this version has been already released. You need to fix this. 
I see this error "!!! Already released version 1.30.0 to SCM.  You can go for publish to push to artifactory." in the Jenkins console log.

**Review by lf12345 — CHANGES_REQUESTED** [review]

File: jenkins/release/Finish (hunk 2 above)
Comment (Ivneetwahi) [reply jenkins/release/Finish:12]: I tried this but didn't worked.

File: jenkins/release/Finish (hunk 1 above)
Comment (Ivneetwahi) [reply jenkins/release/Finish]: Yes, changed.

File: jenkins/hotfix/Build (hunk 3 above)
Comment (Ivneetwahi) [reply jenkins/hotfix/Build]: Removed branch

File: lib/awesome_gem/version.rb (hunk 4 above)
Comment (Ivneetwahi) [reply lib/awesome_gem/version.rb]: Changed version

File: jenkins/release/Build [hunk 5]
```diff
@@ -1,5 +1,17 @@
...
-  gemPipelines.releaseBuild(name: 'awesome_gem',autoRelease: true)
+node 
+{
+  try
+  {
+    gemPipelines.releaseBuild(name: 'awesome_gem',autoRelease: true)
+  }
+  catch(err)
+  {
+  	if (env.mailAlert)
```
Comment (lf12345) [inline jenkins/release/Build:10]: Please add a message saying we are sending out a failure email.

File: jenkins/release/Finish [hunk 6]
```diff
@@ -1,6 +1,17 @@
...
+node
+{
+  try
+  {
+    gemPipelines.releaseFinish(name: 'awesome_gem')
+  }
+  catch(err) 
+  {
+  	if (env.mailAlert)
+	    {
```
Comment (lf12345) [inline jenkins/release/Finish:11]: Please add a message saying we are sending out a failure email.

File: jenkins/release/Start [hunk 7]
```diff
@@ -1,5 +1,17 @@
...
+node 
+{
+  try 
+	{
+    gemPipelines.releaseStart(name: 'awesome_gem',autoRelease: true)
+  }
+  catch(err)
+  {
+  	if (env.mailAlert)
+	    {
```
Comment (lf12345) [inline jenkins/release/Start:11]: Please add a message saying we are sending out a failure email.

**Review by lf12345 — CHANGES_REQUESTED** [review]

**Review by lf12345 — APPROVED** [review]

**Pull request closed without merging**